docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
     build:
       context: .
       args:
-        NODE_ENV: development
+        NODE_ENV: ${NODE_ENV-development}
```

Comment:
Does the variable want setting when the value has is either unset or empty or just when it is unset?
See docs for more info: 
https://docs.docker.com/compose/compose-file/compose-file-v2/#variable-substitution